resources/keywords/deploy_instance_dev.resource

```diff
@@ -78,7 +78,9 @@ User Filters And Selects The Created Database Instance
 Instance Is Available In Selection Table
     ${instFound}=    Run Keyword And Return Status    Instance Can Be Selected
     Run Keyword If    not ${instFound}
+    ...    Run Keywords
     ...    Reload Page
+    ...    Fail
```

Comment:
The keyword _Instance Is Available In Selection Table_ considered to be passed regardless of the keyword _Instance Can Be Selected_ status. This is due to, the keyword status has been handled by the variable ${instFound}. 
Introduced _Fail_ keyword on line 83, therefore if the instance targeted is not available then the page reloaded and the keyword marked as failed. With the failed status, this keyword would be retried for 5x.